doc/classes/AABB.xml

```diff
@@ -9,9 +9,9 @@
 		[b]Note:[/b] Unlike [Rect2], [AABB] does not have a variant that uses integer coordinates.
 	</description>
 	<tutorials>
-		<link title="Math tutorial index">https://docs.godotengine.org/en/latest/tutorials/math/index.html</link>
-		<link title="Vector math">https://docs.godotengine.org/en/latest/tutorials/math/vector_math.html</link>
-		<link title="Advanced vector math">https://docs.godotengine.org/en/latest/tutorials/math/vectors_advanced.html</link>
+		<link title="Math tutorial index">https://docs.godotengine.org/en/$DOC_VERSION/tutorials/math/index.html</link>
```

Comment:
Maybe we could replace the whole of `https://docs.godotengine.org/en/<version>` with `$GODOT_DOCS` instead? Less chars, less hardcoding of URL, and keeps the possibility open for pointing at localized docs when relevant, if we decide to do so.

Or could even be `godot://` if we want to make it like a URI.

Reply:
I would avoid using a scheme like that in case we actually might need it for something (like opening projects with the editor maybe?)

Reply:
OK for not using a scheme, but I still think replacing the whole constant URL prefix with `$GODOT_DOCS` or similar would be beneficial.

Reply:
Bump :)

Reply:
Monthly ping :)